written_responses.md

```diff
@@ -1,5 +1,11 @@
 1. What is a regular expression and how does it work?
```

Comment:
Very concise answers Michael. I like that you're leaving room to expand on your answers if the interviewer were to ask. Your writing will also work well with both technical and non-technical interviewers 👍